src/node/stage/approved.rs

```diff
@@ -2430,15 +2415,40 @@ impl Approved {
                 VoteTheirKnowledge { prefix, key_index } => {
                     self.cast_unordered_vote(core, Vote::TheirKnowledge { prefix, key_index })?
                 }
-                VoteSendNeighbourInfo { dst, nonce } => {
-                    self.cast_ordered_vote(AccumulatingEvent::SendNeighbourInfo { dst, nonce })
+                SendNeighbourInfo { dst, nonce } => {
+                    self.send_neighbour_info(core, dst, nonce, None)?
```

Comment:
The `dst_key` should be obtained from `shared_state.sections.key_by_name(dst)`.

Reply:
I remember it was previous doing so however having issue, will roll back to check what was the issue (or maybe no longer have)